### ISSUE TITLE
allow the collapse of columns 

### ISSUE THREAD

**Issue by arademaker** [odanoburu/conllu-mode#2]
long columns such as FEATS could be collapsed for facilitate the analysis of the dependencies.

**Comment by odanoburu** [comment]
no idea how to do this, but https://www.gnu.org/software/emacs/manual/html_node/emacs/Selective-Display.html and https://www.gnu.org/software/emacs/manual/html_node/emacs/Outline-Mode.html implement similar (not on columns) functionality

**Comment by arademaker** [comment]
It looks like all modes that implement this function use the basic Emacs https://www.gnu.org/software/emacs/manual/html_node/elisp/Invisible-Text.html#Invisible-Text capability.

**Comment by odanoburu** [comment]
@arademaker I seem to have got it to work in branch `hide-cols`. would you mind testing it?

**Comment by odanoburu** [comment]
hum, actually there is at least one problem -- when editing right next to the collapsed field the inserted text becomes part of it. I think I know how to handle this, let's see...

if you don't do that everything seems fine, if you want to check it out!

**Comment by odanoburu** [comment]
seems ok now 0a42657

**Comment by arademaker** [comment]
The hide/show is working, but not playing well with the align of columns. A hide field would not interfere in the alignment.

**Comment by odanoburu** [comment]
@arademaker indeed. I hoped it would be simply a miscalculation of column widths when they were invisible, but I just checked, and it is not so. it must be something in the overlays, but I wouldn't know what, as I'm not familiar with them yet...
